.github/workflows/ci.yml

```
@@ -116,14 +132,16 @@ jobs:
           profile: minimal
           override: true
 
-      - name: Check
-        uses: actions-rs/cargo@v1
-        with:
-          command: check
-          args: ${{ matrix.features }} --all --examples
+      - name: Test (Windows)
+        if: matrix.os == 'windows-latest'
+        shell: cmd
+        run: |
+          call "C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/VC/Auxiliary/Build/vcvars64.bat"
+          cargo test --all
```

```suggestion
          cargo test ${{ matrix.features }} --all
```

we lost `codec-aom` tests

---

This seems reasonable